bucket/sql-workbench-dev.json

```
@@ -38,11 +38,31 @@
             ]
         }
     },
+    "persist": [
+        "ext",
+        "sqlwbconsole64.ini",
+        "sqlwbconsole.ini",
+        "SQLWorkbench64.ini",
+        "SQLWorkbench.ini"
+    ],
+    "pre_install": [
+        "if(!(Test-Path \"$dir/sqlwbconsole64.ini\")) { Add-Content \"$dir/sqlwbconsole64.ini\" $null }",
```

User cross check from persist_dir. No need to use add-content. Rather use new-item (no opening and closing file)
Same for all lines bellow
```suggestion
        "if(!(Test-Path \"$persist_dir\\sqlwbconsole64.ini\")) { New-Item \"$dir\\sqlwbconsole64.ini\" -Type File }",
```